src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceStringParts = sourceString.split(';');
+  const strWithoutSpaces = sourceStringParts.map((str) => str.trim());
```

Comment:
u don't need an extra iteration since you're already using the `reduce` method. Below I'll show u how u can do this